### ISSUE TITLE
Distinct entries

### ISSUE THREAD

**Issue by sarita-dc** [mizzao/meteor-autocomplete#114]
I want to fetch distinct entries from the field. Is there any option where I can avoid duplicate entries?

**Comment by developeron29** [comment]
+1

**Comment by Yonben** [comment]
+1

**Comment by mizzao** [comment]
This is best done in the logic that populates the collection you give to autocomplete, and not handled by the package itself.

Perhaps you can try publishing a [distinct](https://docs.mongodb.org/v3.0/reference/method/db.collection.distinct/) query from the server, for example.

**Comment by Yonben** [comment]
@mizzao Maybe there's a way I don't see because of my lack of experience, but collection.distinct() returns an array not a cursor, so I can't integrate it with the auto-complete, as it asks for a collection.

**Comment by mizzao** [comment]
Yeah, you'd have to do some more legwork to get it from the raw Mongo query, over a publication, into a collection on the client :)

**Comment by Yonben** [comment]
I see, thanks a lot !

2016-02-10 17:56 GMT+02:00 Andrew Mao <notifications@github.com>:
